### PR TITLE
5637 time based log rollover to staging

### PR DESCRIPTION
**Pull request by dmuelle**
#5637

### CODE REVIEW

File: modules/ROOT/pages/access-logging.adoc [hunk 1]
```diff
@@ -17,22 +17,22 @@ If you do not specify attributes, the defaults are used. To see a list of the de
 
 == HTTP access log settings
 
-* Using a common log
-+
-To enable logging for multiple endpoints with common settings, include `httpAccessLogging` as a top-level element in your `server.xml` file, and then reference it from multiple `httpEndpoint` elements, as shown in the following example:
+You can configure HTTP access log settings either for multiple endpoints that share common log settings or for individual endpoints.
 
+Settings for a common log::
+To enable logging for multiple endpoints with common settings, include `httpAccessLogging` as a top-level element in your `server.xml` file, and then reference it from multiple `httpEndpoint` elements, as shown in the following example:
```
Comment (ramkumar-k-9286) [inline modules/ROOT/pages/access-logging.adoc]:
```suggestion
To enable logging for multiple endpoints with common settings, include `httpAccessLogging` as a top-level element in your `server.xml` file. Reference this element from multiple `httpEndpoint` elements, as shown in the following example.
```

Comment (ramkumar-k-9286) [inline modules/ROOT/pages/access-logging.adoc]: Re-phrased the sentence as Acrolinx picked up this for being too long. Also, removed : and replaced with a period.

File: modules/ROOT/pages/access-logging.adoc [hunk 2]
```diff
@@ -17,22 +17,22 @@ If you do not specify attributes, the defaults are used. To see a list of the de
...
 ----
 <httpAccessLogging id="accessLogging"/>
 <httpEndpoint id="defaultHttpEndpoint" accessLoggingRef="accessLogging" httpPort="9080" httpsPort="9443"/>
 <httpEndpoint id="otherHttpEndpoint" accessLoggingRef="accessLogging" httpPort="9081" httpsPort="9444"/>
 ----
 
-* Using distinct logs for each endpoint
-+
+Settings for distinct logs for each endpoint::
 To enable logging for individual endpoints, use an `accessLogging` child element and specify a file path that does not conflict with other logs, as shown in the following example:
```
Comment (ramkumar-k-9286) [inline modules/ROOT/pages/access-logging.adoc]:
```suggestion
To enable logging for individual endpoints, use an `accessLogging` child element and specify a file path that does not conflict with other logs, as shown in the following example.
```

File: modules/ROOT/pages/access-logging.adoc [hunk 3]
```diff
@@ -41,16 +41,19 @@ To enable logging for individual endpoints, use an `accessLogging` child element
 
 == HTTP access log format
 
-Use the `accessLogFormat` property to specify the information  and format you want to include in the NCSA access log for an HTTP transport channel. The value for this property is a space-separated list of options.
+Use the `accessLogFormat` property to specify the information and format you want to include in the NCSA access log for an HTTP transport channel. The value for this property is a space-separated list of options.
```
Comment (ramkumar-k-9286) [inline modules/ROOT/pages/access-logging.adoc:44]: Acrolinx picked this up for being too long.

File: modules/ROOT/pages/access-logging.adoc [hunk 4]
```diff
@@ -41,16 +41,19 @@ To enable logging for individual endpoints, use an `accessLogging` child element
...
 
-This log format string is specified by using the `logFormat` attribute of `httpAcccessLogging` or `accessLogging` elements in the `server.xml` file, as shown in the following examples:
+This log format string is specified by using the `logFormat` attribute of `httpAcccessLogging` or `accessLogging` elements in the `server.xml` file, as shown in the following examples.
```
Comment (ramkumar-k-9286) [inline modules/ROOT/pages/access-logging.adoc]: Acrolinx picked this up for being too long.

File: modules/ROOT/pages/access-logging.adoc [hunk 5]
```diff
@@ -41,16 +41,19 @@ To enable logging for individual endpoints, use an `accessLogging` child element
...
 
+In the following `server.xml` file example, the `logformat` attribute for the `httpAcccessLogging` element specifies the log format string for multiple endpoints that reference the `accessLogging` configuration ID.
```
Comment (ramkumar-k-9286) [inline modules/ROOT/pages/access-logging.adoc]: Acrolinx picked this up for being too long.

File: modules/ROOT/pages/access-logging.adoc (hunk 3 above)
Comment (dmuelle) [reply modules/ROOT/pages/access-logging.adoc:44]: I see these sentences as being 26 and 11 words long- not too long.

File: modules/ROOT/pages/access-logging.adoc (hunk 4 above)
Comment (dmuelle) [inline modules/ROOT/pages/access-logging.adoc]:
```suggestion
Specify the log format string with the `logFormat` attribute of `httpAcccessLogging` or `accessLogging` elements in the `server.xml` file, as shown in the following examples.
```

File: modules/ROOT/pages/access-logging.adoc [hunk 6]
```diff
@@ -41,16 +41,19 @@ To enable logging for individual endpoints, use an `accessLogging` child element
 
 == HTTP access log format
 
-Use the `accessLogFormat` property to specify the information  and format you want to include in the NCSA access log for an HTTP transport channel. The value for this property is a space-separated list of options.
+Use the `accessLogFormat` property to specify the information and format you want to include in the NCSA access log for an HTTP transport channel. The value for this property is a space-separated list of options.
 
-This log format string is specified by using the `logFormat` attribute of `httpAcccessLogging` or `accessLogging` elements in the `server.xml` file, as shown in the following examples:
+Specify the log format string with the `logFormat` attribute of `httpAcccessLogging` or `accessLogging` elements in the `server.xml` file, as shown in the following examples.
 
+In the following `server.xml` file example, the `logformat` attribute for the `httpAcccessLogging` element specifies the log format string for multiple endpoints that reference the `accessLogging` configuration ID.
```
Comment (dmuelle) [inline modules/ROOT/pages/access-logging.adoc]:
```suggestion
In the following `server.xml` file example, The `logformat` attribute for the `httpAcccessLogging` element specifies the log format string. Multiple endpoints can specify this string by referencing the `accessLogging` configuration ID.
```

**Pull request merged**